airbyte-cdk/python/unit_tests/sources/declarative/requesters/error_handlers/test_composite_error_handler.py

```diff
@@ -108,3 +110,58 @@ def test_composite_error_handler_no_handlers():
         assert False
     except ValueError:
         pass
+
+
+def test_error_handler_compatibility_simple():
+    status_code = 403
+    expected_action = ResponseAction.IGNORE
+    response_mock = create_response(status_code)
+    default_error_handler = DefaultErrorHandler(config={}, parameters={}, response_filters=[
+        HttpResponseFilter(action=ResponseAction.IGNORE, http_codes=set([403]), config={}, parameters={})])
+    composite_error_handler = CompositeErrorHandler(error_handlers=[
+        DefaultErrorHandler(
+            response_filters=[HttpResponseFilter(action=ResponseAction.IGNORE, http_codes=set([403]), parameters={}, config={})],
+            parameters={}, config={})
+    ], parameters={})
+    actual_action = default_error_handler.interpret_response(response_mock).action
+    assert actual_action == expected_action
+    assert actual_action == composite_error_handler.interpret_response(response_mock).action
```

Comment:
These last 3 statements are written a bit inconsistently. And it feels weird to be comparing `actual_action` against another actual result.  It would make sense as either:
```
assert expected_action == default_error_handler.interpret_response(response_mock).action
assert expected_action == composite_error_handler.interpret_response(response_mock).action
```

Or:
```
actual_action = default_error_handler.interpret_response(response_mock).action
assert actual_action == expected_action
actual_action = composite_error_handler.interpret_response(response_mock).action
assert actual_action == expected_action
```

Reply:
agreed. fixed